integration_tests/src/main/python/data_gen.py

```diff
@@ -868,9 +868,9 @@ def gen_scalars_for_sql(data_gen, count, seed=0, force_no_nulls=False):
 
 boolean_gens = [boolean_gen]
 
-single_level_array_gens = [ArrayGen(sub_gen) for sub_gen in all_basic_gens + decimal_gens + [null_gen]]
```

Comment:
FYI null_gen was already a part of all_basic_gens so we would double up a few tests.